scripts/delete_resources.sh

```diff
@@ -1,8 +1,24 @@
 #!/bin/bash
 
-#Get resource name from tfvarslocal && change name to make more sense in this context
-PRODUCT_NAME=$(grep ENV_PRODUCT <../config/config.yaml | cut -d: -f2 | tr -d ' "')
+#Get resource name from tfvars file
+PRODUCT_NAME=$(grep ENV_PRODUCT <../config/product.yaml | cut -d: -f2 | tr -d ' "')
+if [[ -z "$PRODUCT_NAME" || ! "$PRODUCT_NAME" =~ ^(rke2|k3s)$ ]]; then
+  echo "Wrong or empty product name found in product.yaml file for: $PRODUCT_NAME"
+  exit 1
+fi
+
 RESOURCE_NAME=$(grep resource_name <../config/"$PRODUCT_NAME".tfvars | cut -d= -f2 | tr -d ' "')
+if [[ -z "$RESOURCE_NAME" ]]; then
+  echo "No resource name found for: $PRODUCT_NAME.tfvars file"
+  exit 1
+fi
+
+if [[ "$RESOURCE_NAME" != *"$USER"* ]]; then
+  printf "Wrong tfvars file found for: %s\nThe resource name should have your username in it\nUser found: %s \
+  \nRESOURCE_NAME found: %s" "$PRODUCT_NAME" "$USER" "$RESOURCE_NAME"
+  exit 1
+fi
+
```

Comment:
I'm a big fan of these changes overall! Thank you 🙌 

Just one question --  do we want `RESOURCE_NAME` to be so tied to `USER`? For example, my `USER=maxross`, but my `RESOURCE_NAME` is often just `max` or `maxr`

Reply:
if the person plan to use this script somewhere or somehow in the future, i think we should "force" that yes
if u just use jenkins jobs for instance, no need to. 

and if so i can add that on the README task as well , to include this info

Reply:
as per discussion now in standup , this is not a mandatory validaton, its just an extra layer of it .

we can delete that and all the validations necessary it will still be good and enough

Reply:
As I have mentioned, I am hesitant to add this check to restrict username to be used in the resource name as I am aware that no one really uses the entire username on resources.. also concerned on the security aspect of using username in resource allocation

Reply:
@rancher-max  any more thoughts here? what u think ?

Reply:
Also I think we should create a separate job in Jenkins to delete resources as we have currently in rancher-ha folder and utilize this script to do similar task

Reply:
I agree that it would be nice to have a separate job in Jenkins to delete resources as MD mentioned.

I do think that this extra layer of validation for USER should be deleted as it really limits us on naming, and people with longer names might even be prevented entirely from create loadbalancers since they have a 32 character limit IIRC. 

I like the idea of having another sanity check though. I think asking for user input could even be good here, since delete can be catastrophic. Something like:
```
printf "This is going to delete all AWS resources with the prefix %s. Continue (y/n)?" "$RESOURCE_NAME"
read  REPLY
if [[ $REPLY =~ ^[Yy]$ ]]
then
  # do delete logic
else
  # print something about not getting confirmation and exiting.
fi
```
Then to run in Jenkins, maybe have it run this script and automatically continue with `echo y | script.sh` or using `yes`.

Reply:
i have deleted this extra validation
and added this

Reply:
LGTM on this - thank you!